### DISCUSSION TITLE
I'm having trouble getting camera2 to display the preview in bs 1.32.2

### DISCUSSION THREAD

**Discussion by ACMDatabase** [kinsi55/CS_BeatSaber_Camera2#111] [Q&A]
I tried upgrading beat saber to 1.32.2 by doing a fresh install and this happens, the preview above the camera is only visible from the back and its just a blinking white and gray screen.
https://drive.google.com/file/d/1rj6C81jv_6fDD6MbaO-cj0SlNBlXKpSO/view?usp=sharing

Any idea why this might be happening?

**Comment by kinsi55** [comment]
With 1.29.4 they made some changes for how Shaders work - The ingame preview here uses a custom shader of mine that I havent explicitly checked actually works since then, I would've expected somebody else to notice this by now tho lol. I'll have a look, thanks for reporting

**Comment by ACMDatabase** [comment]
it seems I wrote the wrong version, it's 1.34.2 that I meant. I'm not sure if it's my Beat Saber or the plugin.

**Comment by kinsi55** [comment]
Will be fixed in next version
